app/src/main/java/com/github/mobile/util/SourceEditor.java

```diff
@@ -166,11 +167,14 @@ public SourceEditor setSource(final String name, final String content, final boo
     }
 
     private void loadSource() {
-        if (name != null && content != null)
+        if (name != null && content != null) {
             if (markdown)
                 view.loadDataWithBaseURL(null, content, "text/html", CHARSET_UTF8, null);
-            else
+            else {
+                view.getSettings().setUseWideViewPort(true);
```

Comment:
This doesn't work if you change the "show markdown" option. Better do something like this:
```
            if (markdown) {
                view.loadDataWithBaseURL(null, content, "text/html", CHARSET_UTF8, null);
                view.getSettings().setUseWideViewPort(false);
            } else {
                view.loadUrl(URL_PAGE);
                view.getSettings().setUseWideViewPort(true);
            }
```